bin/fuel-core/src/cli/run/relayer.rs

```diff
@@ -31,10 +28,6 @@ pub struct RelayerArgs {
     #[arg(long = "relayer-v2-listening-contracts", value_parser = parse_h160, env)]
     pub eth_v2_listening_contracts: Vec<H160>,
 
-    /// Number of da block that the contract is deployed at.
-    #[clap(long = "relayer-da-deploy-height", default_value_t = Config::DEFAULT_DA_DEPLOY_HEIGHT, env)]
-    pub da_deploy_height: u64,
```

Comment:
We still need a way to pass the block height from which we need to start requesting ethereum blocks. Otherwise we will start from zero and it could take a lot of time to catch up with the current block height.